.github/workflows/functional-test.yml

```diff
@@ -43,21 +47,29 @@ jobs:
         model: 'iPhone 15 Plus'
         os_version: '17.4'
```

Comment:
It makes sense to extract these values and keep them in a single place, for example  env variables
The reason for it is that when we need to change iOS version this also usually requires to change phone model and macOS version

Reply:
Yes, similar thing will occur in Android as well, I believe. So firsly I move Azure to GHA without such definition and will do for Android and iOS both later. Then perhaps it would make sense to modify python code side as well to refer to them.